RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -764,6 +764,8 @@ boolean auto_pillKeeperFreeUseAvailable();	//Defined in autoscend/iotms/auto_mr2
 boolean auto_pillKeeperAvailable();			//Defined in autoscend/iotms/auto_mr2019.ash
 boolean auto_pillKeeper(int pill);			//Defined in autoscend/iotms/auto_mr2019.ash
 boolean auto_pillKeeper(string pill);		//Defined in autoscend/iotms/auto_mr2019.ash
+boolean changeSnapperPhylum(phylum toChange);
+boolean snapperPreAdventure(location loc);
```

Comment:
Maybe add `auto_` at the start of these function names? It's not like we're completely consistent on that though, so... whatever.

Reply:
done.